### ISSUE TITLE
One test fails

### ISSUE THREAD

**Issue by yurivict** [jewettaij/moltemplate#86]
Hello,

The FreeBSD port runs this script as a testsuite:
```
        @cd ${WRKSRC} && \
                ([ -e shunit2 ] || (${MKDIR} tests/shunit2 && ${RM} tests/shunit2/shunit2 && ${LN} -s ${LOCALBASE}/bin/shunit2 tests/shunit2/shunit2)) && \
                ${SETENV} ${TEST_ENV} bash tests/test_read_coords_pdb.sh && \
                ${SETENV} ${TEST_ENV} bash tests/test_ltemplify.sh && \
                ${SETENV} ${TEST_ENV} bash tests/test_oplsaa.sh && \
                ${SETENV} ${TEST_ENV} bash tests/test_compass.sh && \
                ${SETENV} ${TEST_ENV} python3 tests/test_genpoly_lt.py
```

In version 2.20.17 it fails like this in the middle:
```
/disk-samsung/freebsd-ports/science/py-moltemplate/work-py39/stage/usr/local/bin/cleanup_moltemplate.sh: line 227: /disk-samsung/freebsd-ports/science/py-moltemplate/work-py39/stage/usr/local/bin/scripts/moltemplate.sh: No such file or directory
ERROR: cleanup_moltemplate.sh: unable to convert the simplified LT file to LAMMPS files
ASSERT:cleanup_moltemplate.sh failed: system.data missing impropers

Ran 1 test.

FAILED (failures=2)
```

**Comment by jewettaij** [comment]
Hi Yuri.  My apologies for my slow reply!  I began a challenging new job and it has been very time consuming at the beginning.  I admit I don't have much time for moltemplate these days.  But I haven't abandoned moltemplate.  I just work on it infrequently.
  I confess that when I first read your error report, I thought it was something specific to freebsd, because the unit-tests were passing in ubuntu linux.  It would have been more polite for me to at least reply to your post.  But when PhnRvTjN reported a [similar bug](https://github.com/jewettaij/moltemplate/issues/88), then I realized it was something more general.
   Thank you both for your reports!
 It looks like a pip (packaging) issue, but PhnRvTjN hinted at a workaround solution.  I will fix this in the next 24 hours.
